2021-10-15 15:11:50,915 INFO    MainThread:76476 [wandb_setup.py:_flush():69] setting env: {}
2021-10-15 15:11:50,915 INFO    MainThread:76476 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-15 15:11:50,915 INFO    MainThread:76476 [wandb_init.py:_log_setup():342] Logging user logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch-Lightning/Intel-Image-Classification-Learning-PyTorch-Lightning/wandb/run-20211015_151150-39gl57q0/logs/debug.log
2021-10-15 15:11:50,915 INFO    MainThread:76476 [wandb_init.py:_log_setup():343] Logging internal logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch-Lightning/Intel-Image-Classification-Learning-PyTorch-Lightning/wandb/run-20211015_151150-39gl57q0/logs/debug-internal.log
2021-10-15 15:11:50,916 INFO    MainThread:76476 [wandb_init.py:_jupyter_setup():294] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f3cb1c44f50>
2021-10-15 15:11:50,916 INFO    MainThread:76476 [wandb_init.py:init():375] calling init triggers
2021-10-15 15:11:50,916 INFO    MainThread:76476 [wandb_init.py:init():382] wandb.init called with sweep_config: {}
config: {}
2021-10-15 15:11:50,916 INFO    MainThread:76476 [wandb_init.py:init():424] starting backend
2021-10-15 15:11:50,916 INFO    MainThread:76476 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-15 15:11:50,929 INFO    MainThread:76476 [backend.py:ensure_launched():135] starting backend process...
2021-10-15 15:11:50,940 INFO    MainThread:76476 [backend.py:ensure_launched():140] started backend process with pid: 99983
2021-10-15 15:11:50,941 INFO    MainThread:76476 [wandb_init.py:init():429] backend started and connected
2021-10-15 15:11:50,992 INFO    MainThread:76476 [wandb_run.py:_label_probe_notebook():815] probe notebook
2021-10-15 15:11:50,993 INFO    MainThread:76476 [wandb_init.py:init():477] updated telemetry
2021-10-15 15:11:50,993 INFO    MainThread:76476 [wandb_init.py:init():500] communicating current version
2021-10-15 15:11:51,866 INFO    MainThread:76476 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-10-15 15:11:51,866 INFO    MainThread:76476 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-10-15 15:11:52,368 INFO    MainThread:76476 [wandb_init.py:init():540] starting run threads in backend
2021-10-15 15:11:57,377 INFO    MainThread:76476 [wandb_run.py:_console_start():1601] atexit reg
2021-10-15 15:11:57,378 INFO    MainThread:76476 [wandb_run.py:_redirect():1475] redirect: SettingsConsole.WRAP
2021-10-15 15:11:57,378 INFO    MainThread:76476 [wandb_run.py:_redirect():1512] Wrapping output streams.
2021-10-15 15:11:57,379 INFO    MainThread:76476 [wandb_run.py:_redirect():1536] Redirects installed.
2021-10-15 15:11:57,379 INFO    MainThread:76476 [wandb_init.py:init():565] run started, returning control to user process
2021-10-15 15:11:57,409 INFO    MainThread:76476 [wandb_init.py:_pause_backend():256] pausing backend
2021-10-15 15:11:57,409 INFO    MainThread:76476 [jupyter.py:save_ipynb():301] looking for notebook: 00.ipynb
2021-10-15 15:11:57,912 INFO    MainThread:76476 [wandb_init.py:_pause_backend():260] saved code: <wandb.sdk.wandb_artifacts.Artifact object at 0x7f3cb1c40f90>
2021-10-15 15:12:07,905 INFO    MainThread:76476 [wandb_init.py:_resume_backend():265] resuming backend
2021-10-15 15:12:07,907 INFO    MainThread:76476 [wandb_init.py:_pause_backend():256] pausing backend
2021-10-15 15:12:07,907 INFO    MainThread:76476 [jupyter.py:save_ipynb():301] looking for notebook: 00.ipynb
2021-10-15 15:12:08,417 INFO    MainThread:76476 [wandb_init.py:_pause_backend():260] saved code: <wandb.sdk.wandb_artifacts.Artifact object at 0x7f3cb2765450>
2021-10-15 15:12:08,421 INFO    MainThread:76476 [wandb_init.py:_resume_backend():265] resuming backend
2021-10-15 15:12:08,428 INFO    MainThread:76476 [wandb_setup.py:_flush():69] setting env: {}
2021-10-15 15:12:08,428 INFO    MainThread:76476 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-15 15:12:08,428 INFO    MainThread:76476 [wandb_init.py:_log_setup():342] Logging user logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch-Lightning/Intel-Image-Classification-Learning-PyTorch-Lightning/wandb/run-20211015_151208-24t4hvnk/logs/debug.log
2021-10-15 15:12:08,428 INFO    MainThread:76476 [wandb_init.py:_log_setup():343] Logging internal logs to /home/indika/Programming/Projects/Python/Artifical-Intelligence/PyTorch-Lightning/Intel-Image-Classification-Learning-PyTorch-Lightning/wandb/run-20211015_151208-24t4hvnk/logs/debug-internal.log
2021-10-15 15:12:08,428 INFO    MainThread:76476 [wandb_init.py:init():375] calling init triggers
2021-10-15 15:12:08,428 INFO    MainThread:76476 [wandb_init.py:init():382] wandb.init called with sweep_config: {}
config: {}
2021-10-15 15:12:08,428 INFO    MainThread:76476 [wandb_init.py:init():397] re-initializing run, found existing run on stack: 39gl57q0
2021-10-15 15:12:08,429 INFO    MainThread:76476 [wandb_run.py:finish():1271] finishing run ranuga-d/Intel-Image-Classification-Learning-PyTorch-Lightning/39gl57q0
